docs/platform/concepts/support-case-and-incident-investigations.rst

```diff
@@ -0,0 +1,45 @@
+Support case and incident investigations
+========================================
+
+Please provide the following information to the Aiven support team
```

Comment:
Do we need to provide a template like this here since there is a support center within Console where tickets can be created that prompt users for some of this information?

Reply:
We have a how to article on creating tickets here: https://docs.aiven.io/docs/platform/howto/project-support-center

Reply:
@staceysalamon-aiven I think that with the introduction of Support Center this PR has become less relevant. That said, it might still offer a template that helps with providing the right information quickly. We could link it to the how to article you linked.

Reply:
We can add some notes in the "enter a description" step of the how-to article, but maybe just service name and affected functionality? Users are already prompted to select a project and service and I think it is unlikely anyone will come to the docs to copy/paste this template into the ticket description.

If that sounds good to you, I can close this PR, make the necessary changes to the other article, and open a PR for you to review.